fizzbuzz問題を解く

munazo opened the pull request:
fizzbuzz問題を解きました。

```
- テストの実行環境構築
- fizzbuzz問題を解く
- テストを書く
- README更新
```

よろしくお願いします。

Commit:
close #1 : テストが実行出来るような環境をつくる

Commit:
close #2 : fizzbuzz問題のプログラムを書く

Commit:
close #3 : README書く

Commit:
close #5 : Ruby2からは"# encoding: utf-8"が不要

Commit:
close #6 : Specのテストケースで数字がそのまま出力されるケースもあったほうがよい

Commit:
close #7 : lib/fizzbuzzer.rbもguardの対象に含める

+ Guardfileを整理しました。

Commit:
close #8 : メソッド名の末尾に「!」がある場合は破壊的メソッドのほうがよいと思います

Commit:
fixed fizzbuzzer_spec.rb : メソッド名を修正

参考 : メソッド名の末尾に「!」がある場合は破壊的メソッドのほうがよいと思います

a-munakata commented:
@hajime-takubo レビューありがとうございます。
issueに対応したので、ご確認お願いします。

hajime-takubo commented:
なるほど。ありがとうございます。

ちなみに、次はrubocopの導入と
rspec -f dで見やすく出力されるようにspecファイルを修正するというissueをあげる予定です

a-munakata commented:
了解です。
こちらはいったんMergeと言う形で良いでしょうか？

hajime-takubo commented:
はい。
今コメントしたものは次回へのcommitでお願いします

hajime-takubo commented:
あとはissueの修正が完了していることの確認が終わりました。

Commit:
数値をそのまま返すメソッドを追加 / テスト修正

"3で割り切れる場合"、"5で割り切れる場合"、"15で割り切れる場合"にも
expect( model ).not_toがあったほうがよいと思います。
数値をそのまま返すメソッドを追加して、spec側でそれを利用するとよいと思います。

a-munakata commented:
> あとはissueの修正が完了していることの確認が終わりました。

ありがとうございます。

a-munakata commented:
> 今コメントしたものは次回へのcommitでお願いします

こちら、対応しましたので、一旦マージします。
その他issueは別のpull requestを送ります。
